packages/zengine-ui-react/src/util/Form.tsx

```diff
@@ -82,32 +85,32 @@ function ZengineUIForm (props: FormProps): React.ReactElement {
             </div>) : null}
 
             {/* If we're showing either a submit or a reset button add a "form-actions" wrapper for them */}
-            {(showSubmit || showReset) && (
+            {(showSubmit === true || showReset === true) && (
               <div className="form-actions d-flex align-items-center">
-                {showSubmit && (
+                {showSubmit === true && (
                   <Button
                     type="submit"
                     theme="primary"
                     aria-label={labelSubmit}
-                    disabled={!touched || isSubmitting || !isValid}
-                    classes='mr-2'
+                    disabled={isSubmitting || !isValid || isEmpty(touched)}
+                    classes="mr-2"
                   >
                     {labelSubmit !== undefined ? labelSubmit : ''}
                   </Button>
                 )}
 
-                {showReset && dirty && (
+                {showReset === true && dirty && (
```

Comment:
what weirdness are we doing that would cause this boolean to be truthy but not `true`?

Reply:
@tehpsalmist nothing that's just typescript being typescript.  that error crept up in linting after I marked those props in the interface as optional so maybe the fact that it can be undefined means ts doesnt like it?  even though it in defaultProps... weird but  i've seen this multiple times with TS not a hug deal just annoying